devlib/utils/android.py

```diff
@@ -20,6 +20,7 @@
 """
```

Comment:
From the commit msg: "[...] a separate thread along with the synchronization that brings."  That should be "the synchronization that **it** brings", right ?

Reply:
Ummmmmmmmmmmmmm I think [it's 'along with the synchronisation [that] that brings' - the first 'that' introduces a relative clause but is sort of optional, the second 'that' is a relative pronoun referring to the separate thread.](https://s-media-cache-ak0.pinimg.com/originals/78/d7/fc/78d7fc8f03a3e1a920e7b195ddce2644.jpg)

Reply:
Clearer example would be "the queen is ashamed to admit _that_ the language _she_ represents is competely absurd" (where the "that" that introduces a relative clause could just be dropped and the "she" is a relative pronoun)

(#nerdsniped)

Reply:
(OK back to work)

Reply:
Fair enough, so much for me trying to be nitpicky on the Queen's language :D